build/exports.ts

```diff
@@ -1,9 +1,10 @@
 import * as ts from "typescript";
-
 let output = new Map();
 
+var glob = require('glob');
+var fileNames = glob.sync(process.argv[2]);
 // Build a program using the set of root file names in fileNames
-let program = ts.createProgram(process.argv.slice(2), {
+let program = ts.createProgram(fileNames, {
```

Comment:
I think it's just used to collect all the exported classes together, because there isn't a real module loader. It's just used at build time, to parse out which classes are exported (and which are static).

Reply:
Leveraged the [NPM rollup plugins](https://github.com/rollup/plugins) to accomplish this. Turns out we weren't the only ones facing this issue 😅